lib/compat/wordpress-6.7/class-gutenberg-rest-templates-controller-6-7.php

```diff
@@ -0,0 +1,375 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Templates_Controller_6_7 class
+ *
+ * @package    gutenberg
+ */
+
+/**
+ * Gutenberg_REST_Templates_Controller_6_7 class
+ *
+ */
+class Gutenberg_REST_Templates_Controller_6_7 extends Gutenberg_REST_Templates_Controller_6_6 {
+	/**
+	 * Retrieves the block type' schema, conforming to JSON Schema.
+	 *
+	 * @since 5.8.0
+	 * @since 5.9.0 Added `'area'`.
+	 * @since 6.7.0 Added `'plugin'`.
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+		if ( $this->schema ) {
+			return $this->add_additional_fields_schema( $this->schema );
+		}
+
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => $this->post_type,
+			'type'       => 'object',
+			'properties' => array(
+				'id'              => array(
+					'description' => __( 'ID of template.' ),
+					'type'        => 'string',
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'slug'            => array(
+					'description' => __( 'Unique slug identifying the template.' ),
+					'type'        => 'string',
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'required'    => true,
+					'minLength'   => 1,
+					'pattern'     => '[a-zA-Z0-9_\%-]+',
+				),
+				'theme'           => array(
+					'description' => __( 'Theme identifier for the template.' ),
+					'type'        => 'string',
+					'context'     => array( 'embed', 'view', 'edit' ),
+				),
+				// @core-merge: Add plugin property to the schema.
+				'plugin'          => array(
+					'description' => __( 'Plugin that registered the template.' ),
```

Comment:
If we end up using this new prop, we should add with `register_rest_field`. There is no need to copy the whole function.

Reply:
TIL, I changed it in 49131f1d9eea0c7c6e956642846994faea89a901.